parachains/runtimes/contracts/contracts-rococo/Cargo.toml

```diff
@@ -52,6 +52,7 @@ pallet-sudo = { git = "https://github.com/paritytech/substrate", default-feature
 pallet-contracts = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "master" }
 pallet-contracts-primitives = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "master" }
 pallet-contracts-rpc-runtime-api = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "master" }
+pallet-contracts-xcm = { git = "https://github.com/paritytech/pallet-contracts-xcm", default-features = false, branch = "master" }
```

Comment:
This will make people creating releases on branches unhappy. Because they will need to add a branch to this repo and change the branches to the appropriate Substrate/Polkadot branches.

This will also make merging stuff that breaks the code in the repo hard. (assuming something in Substrate changed)

Reply:
What else can we do? Move that stuff into the polkadot repo?

Reply:
Or into the Cumulus repo :see_no_evil: